client/components/account-balances/balances-tab-panel.tsx

```diff
@@ -57,6 +58,12 @@ const AccountBalancesTabPanel: React.FC = () => {
 
 	const onTabSelect = ( tabName: BalanceTab[ 'name' ] ) => {
 		setSelectedCurrency( tabName );
+		wcpayTracks.recordEvent(
+			wcpayTracks.events.OVERVIEW_BALANCE_CURRENCY_CLICK,
+			{
+				currency: tabName,
```

Comment:
Suggest wording the action and the property so it's 100% clear that the `currency` is the newly chosen currency, e.g. `wcadmin_wcpay_overview_balances_currency_tab_click selected_currency=USD|EUR|GBP` etc.

Reply:
Addressed here [1dcd9e6](https://github.com/Automattic/woocommerce-payments/pull/6307/commits/1dcd9e602f92ae3ba7424c223455e4ef693b981c).